src/chanjo2/crud/samples.py

```diff
@@ -74,3 +75,11 @@ def create_sample_in_case(db: Session, sample: SampleCreate) -> Optional[SQLSamp
     db.commit()
     db.refresh(db_sample)
     return db_sample
+
+
+def remove_sample(db: Session, sample_name: str) -> int:
+    """Remove a sample with a given name."""
```

Comment:
```suggestion
    """Delete sample with the supplied name."""
```
I guess the sample name is unique so there can only be one?

Reply:
Yes, it's unique: https://github.com/Clinical-Genomics/chanjo2/blob/c79d875295cc01fe048c234a5a3dbc98de80a321/src/chanjo2/models/sql_models.py#L27